onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -415,6 +418,7 @@ void BeamSearchShapeInference(ONNX_NAMESPACE::InferenceContext& ctx) {
   // output 0 (sequences) shape: (batch_size, num_return_sequences, max_length)
   // output 1 (sequences_scores) shape: (batch_size, num_return_sequences)
   // output 2 (scores) shape: (max_length - sequence_length, batch_size, num_beams, vocab_size)
+  // output 3 (cross_attention): shape: (batch_size, num_return_sequences, Layers, Heads, max_length, Frames)
```

Comment:
The code uses sequence_length as last dim. Shall we update the comment here?

Reply:
Here is more conceptual .